config.json

```diff
@@ -1,9 +1,29 @@
 {
+    "discordBot": true,
+    "webserver": false,
     "ports":{
-        "minecraft":25565,
-        "web":80
+        "minecraft":25566, // port for the proxy server
```

Comment:
Hmm ``//`` are no comments in json :/

Reply:
every json is loaded through a sanitizer.